packages/types/src/clerk.ts

```diff
@@ -681,11 +688,17 @@ export type SetActiveParams = {
   session?: ActiveSessionResource | string | null;
 
   /**
-   * The organization resource or organization id (string version) to be set as active in the current session.
+   * @deprecated The organization resource or organization id (string version) to be set as active in the current session.
    * If `null`, the currently active organization is removed as active.
    */
   organization?: OrganizationResource | string | null;
 
+  /**
+   * The organization, identified via slug or ID, to be set as active in the current session.
+   * If `null`, the currently active organization is removed as active.
+   */
+  organizationIdentifier?: OrganizationIdentifier | null;
+
```

Comment:
Here's an idea thanks to @gkats : https://github.com/clerk/clerk_go/pull/7174#discussion_r1687485284

It looks like bapi accepts a string that can be the slug _or_ the org id in other places, and it works fine. If we do that here, we don't need to change the API at all! 

I wouldn't have expected this to work, but as @mzhong9723 pointed out here: https://github.com/clerk/clerk_go/pull/7174#discussion_r1687162721 we have validation logic already that prevents a slug from sharing a name with an org ID. 

It strikes me that there's slightly elevated potential for both user confusion (e.g. a user is trying to specify an org ID that doesn't exist, but a slug that shares the name _does_ exist, so we return that), and bugs on our side (we need to make sure that we always test the string against both the org id _and_ the slug every time). But it still strikes me as the best path forward, seeing as we already have the validation logic and a precedent and it makes for a much smaller change!